### ISSUE TITLE
Pascal case option for export

### ISSUE THREAD

**Issue by lostcart** [RedMadRobot/figma-export#222]
Is it possible to add support for more case options on export? Specific use case is Pascal case support for typography export on Android

**Comment by subdan** [comment]
Yes, it's possible. But unfortunately I don't have time to do it. If you know Swift, then feel free to open a PR.
